config/provider.go

```diff
@@ -85,31 +66,22 @@ func getSharedConfigFiles(configs configs) ([]string, bool, error) {
 	return nil, false, nil
 }
```

Comment:
Did we intentionally squash SharedCredentialFiles into a single field `SharedConfigFiles`?  I thought those were two different fields at one point. I may of missed that.

Reply:
Don't know the history behind this change. It is currently squashed.  They can be separate, but the current usage doesn't require them to be.